dashboard/config/routes.rb

```diff
@@ -534,6 +534,7 @@ module OPS
   get '/dashboardapi/v1/schools/:school_district_id/:school_type', to: 'api/v1/schools#index', defaults: {format: 'json'}
   # We want to allow school search on things like "St. Paul" so we specify the constraint on :q to match anything but a slash
   # See http://guides.rubyonrails.org/routing.html#specifying-constraints
+  get '/dashboardapi/v1/districtsearch/:q/:limit', to: 'api/v1/school_districts#search', defaults: {format: 'json'}, constraints: {q: /[^\/]+/}
```

Comment:
Do you need this constraint? I see it's specified on the next line too, but if I'm reading it correctly it's asking for at least one character that's not a `/`. Isn't that how Rails would parse the path anyway? I don't think this does anything.

I'm fairly certain that, for instance, `districtsearch/foo/bar` would already parse into params `{q: 'foo', limit: 'bar'}`.

Reply:
@drewsamnick can you comment on this comment? This was added after I did the initial school auto-complete.

Reply:
The constraint was put in place for schools to allow matching on a period. By default it won't match and that caused errors when searching for things like "St. Paul"

Reply:
See the comment on line 535-536.

Reply:
Gotcha. Since that is pretty far down in the linked constraint description, let's add a small comment about the purpose too, e.g. "Constrain the q param to allow dots" before that link